eng/Versions.props

```diff
@@ -37,7 +37,7 @@
          but not higher than our minimum dogfoodable Visual Studio version, or else
          the generators we build would load on the command line but not load in IDEs. -->
     <SourceGeneratorMicrosoftCodeAnalysisVersion>4.1.0</SourceGeneratorMicrosoftCodeAnalysisVersion>
-    <MicrosoftILVerificationVersion>7.0.0-alpha.1.22060.1</MicrosoftILVerificationVersion>
+    <MicrosoftILVerificationVersion>7.0.0-preview.1.22076.8</MicrosoftILVerificationVersion>
```

Comment:
ILVerification changed some interfaces after preview1 and roslyn will need to react.